docs/guide/reviewing.md

```diff
@@ -1,16 +1,138 @@
+
+### Review
+
+An examination of a submission. Editors, review coordinators, reviewers, and submitters can participate in a review. Reviews are managed by review coordinators.
+
+![Purple circular icon of a magnifying glass on a wavy sheet of paper, 'Review'](/public/icons/review.svg)
+
+### Overall Comment
+
+This is a comment in which users can comment on the submission broadly or generally. These do not have style criteria associated with them.
+
+![Purple circular icon of a chat bubble titled, 'Overall Comment'](/public/icons/overall-comment.svg)
+
+### Inline Comment
+
+This is a comment in which users can highlight portions of the submission and make comments inline with the content. Users can associate style criteria with their inline comments.
+
+![Purple circular icon of a highlighter in use titled, 'Inline Comment'](/public/icons/inline-comment.svg)
+
+#### Style Criteria
+
+Also called "review criteria." These are defined by a publication. Users making inline comments on submissions can pick from style criteria to associate with their inline comments. Style criteria are not associated with overall comments.
+
+![Purple circular icon of a rectangular list item titled, 'Style Criteria'](/public/icons/style-criteria.svg)
+
+## Roles
+
+### Editor
+
+A publication-relative user who manages a publication.
+
+![Green circular icon of a publication icon on the bust of a person titled, 'Editor'](/public/icons/editor.svg)
+
+### Submitter
+
+A submission-relative user that creates submissions.
+
+Examples of submitters are: authors, coauthors, or creators. Anyone who intends to submit content and have it peer reviewed collaboratively by a publication assumes the role of submitter.
+
+![Blue circular icon of a submission icon on the bust of a person titled, 'Submitter'](/public/icons/submitter.svg)
+
+### Reviewer
+
+A submission-relative user who reviews submissions.
+
+![Purple circular icon of a review icon on the bst of a person titled, 'Review Coordinator'](/public/icons/reviewer.svg)
+
+### Review Coordinator
+
+A submission-relative user who manages a review of a submission.
+
+![Purple circular icon of a review icon on the bst of a person titled, 'Review Coordinator' with a plus symbol and bust of a person](/public/icons/review-coordinator.svg)
+
+## Overview
+
+![Overview of a successful Collaborative Community Review process in Pilcrow featuring 8 steps involving a submitter, an editor, a review coordinator, and a reviewer.](/public/review/overview.svg)
+
+### Step 1
+
+A submitter submits a submission to a publication.
+
+![Diagram of a submitter submitting to a publication](/public/review/step1.svg)
+
+### Step 2
+
+Before any review can take place, an editor of the publication evaluates the submission's fitness for review. If the editor deems the submission worthy of review, the editor proceeds to step 3. Otherwise, the editor can reject the submission or request it for resubmission.
```

Comment:
```suggestion
Before any review can take place, an editor of the publication evaluates the submission's fitness for review. If the editor deems the submission reviewable, the editor proceeds to step 3. Otherwise, the editor can reject the submission or request it for resubmission.
```